python/core/symbology-ng/qgssymbollayerutils.sip

```diff
@@ -501,4 +501,20 @@ class QgsSymbolLayerUtils
       */
     static void mergeScaleDependencies( int mScaleMinDenom, int mScaleMaxDenom, QgsStringMap& props );
 
+    /**
+     * Encodes a reference to a parametric SVG into SLD, as a succession of parametric SVG using URL parameters,
+     * a fallback SVG without parameters, and a final fallback as a mark with the right colors and outline for systems
+     * that cannot do SVG at all
+     * @note added in 3.0
+     */
+    static void parametricSvgToSld( QDomDocument &doc, QDomElement &graphicElem,
```

Comment:
Renaming to fill color, not a problem.
The second I'm not 100% syre, as I (thought I) was just following an established pattern, e.g.:
https://github.com/qgis/QGIS/blob/master/src/core/symbology-ng/qgssymbollayerutils.h#L242
https://github.com/qgis/QGIS/blob/master/src/core/symbology-ng/qgssymbollayerutils.h#L248
https://github.com/qgis/QGIS/blob/master/src/core/symbology-ng/qgssymbollayerutils.h#L257

However I also see smaller helper methods returning the element indeed:
https://github.com/qgis/QGIS/blob/master/src/core/symbology-ng/qgssymbollayerutils.h#L320
https://github.com/qgis/QGIS/blob/master/src/core/symbology-ng/qgssymbollayerutils.h#L323

So I'll follow your advice for this one too